node/views/bmi-calculator/bmi-calculator.ejs

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<head>
+    <meta charset="utf-8">
+    <title>BMI Calculator</title>
+    <link rel="stylesheet" href="./stylesheets/bmi-calc.css">
+</head>
+
+
+<body>
+    <div>
+        <h2>BMI Calculator</h2>
+        <p class="text">Height</p>
+        <input type="text" id="h">
+        <p class="text">Weight</p>
+        <input type="text" id="w">
+        <p id="result"></p>
+        <button id="btn" onClick="BMI()">Calculate</button>
```

Comment:
generally don't mix html and js, use event listener when possible

Reply:
okay